Common/HttpWebRequestWrapperLib/HttpWebRequestWrapper.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Text;
+
+namespace HttpWebRequestWrapperLib
+{
+    /// <summary>
+    /// Wrapper of http web requests and responses that encapsulate GET, PUT POST and DELETE requests
+    /// </summary>
+    public class HttpWebRequestWrapper
+    {
+        public HttpWebRequest httpWebRequest { get; set; }
+        public HttpWebResponse httpWebResponse { get; set; }
+
+        public HttpWebRequestWrapper(string contentType = "application/json")
+        {
+            httpWebRequest.ContentType = contentType;
+        }
+        
+        public string Get(string url, Dictionary<string,string> parameters)
+        {
+            if (string.IsNullOrEmpty(url))
+            {
+                return "Url is null or empty";
+            }
+            if (!(parameters is null) || parameters.Count > 0)
+            {
+                url = getUrlWithParams(url, parameters);
+            }
+            httpWebRequest.Method = "GET";
+            HttpWebRequest.Create(url);
+
+            httpWebResponse = (HttpWebResponse)httpWebRequest.GetResponse();
+            using Stream responseStream = httpWebResponse.GetResponseStream();
+            using StreamReader streamReader = new StreamReader(responseStream, Encoding.UTF8);
```

Comment:
This contradicts our code convention. I think, `var` should be here

Reply:
Where? httpResponse is field of our class, using is special construction that close stream after usage

Reply:
` using StreamReader streamReader = new StreamReader`
I mean, here should be `var` instead `StreamReader` from the left side

Reply:
Oh, okay, I see now. Thanks